editor/js/Sidebar.Project.Image.js

```diff
@@ -69,6 +69,44 @@ function SidebarProjectImage( editor ) {
 	renderButton.setMarginLeft( '120px' );
 	renderButton.onClick( async () => {
 
+		if ( shadingTypeSelect.getValue() === 'realistic' ) {
+
+			let isMaterialsValid = true;
+
+			editor.scene.traverseVisible( ( object ) => {
+
+				if ( object.isMesh ) {
+
+					const materials = Array.isArray( object.material ) ? object.material : [ object.material ];
+
+					for ( let i = 0; i < materials.length; i ++ ) {
+
+						const material = materials[ i ];
+
+						if ( ! material.isMeshStandardMaterial && ! material.isMeshPhysicalMaterial ) {
+
+							isMaterialsValid = false;
+							return;
+
+						}
+
+					}
+
+				}
+
+			} );
+
+			if ( isMaterialsValid === false ) {
+
+				alert( 'REALISTIC Shading: Only MeshStandardMaterial and MeshPhysicalMaterial are supported' ); // TODO: l10n
```

Comment:
Do you mind translating this right away? Otherwise we end up with an additional PR which is an unnecessary fragmentation.

Reply:
ok